xrpl/transaction/main.py

```diff
@@ -40,24 +41,25 @@ def safe_sign_and_submit_transaction(
     if autofill:
         transaction = safe_sign_and_autofill_transaction(transaction, wallet, client)
     else:
-        transaction = safe_sign_transaction(transaction, wallet)
+        transaction = safe_sign_transaction(transaction, wallet, client)
     return submit_transaction(transaction, client)
 
 
 def safe_sign_transaction(
-    transaction: Transaction,
-    wallet: Wallet,
+    transaction: Transaction, wallet: Wallet, client: Client
```

Comment:
This method should not require a `Client` - the point of this method is that it's not sending any requests to the ledger, it's signing the Transaction object entirely offline. This would also be a breaking change and shouldn't be introduced in a minor release.

Reply:
Understood thank you, I removed the client.